cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog/hack/lib/util/golang.sh

```diff
@@ -8,7 +8,7 @@ function os::golang::verify_go_version() {
 
 	local go_version
 	go_version=($(go version))
-	if [[ "${go_version[2]}" != go1.7* ]]; then
+	if [[ "${go_version[2]}" != go1.8* ]]; then
```

Comment:
have the vendored builds been updated to work with go1.8? I know there was fixup work in kube that was needed